frontend/XCITE/Nodes/TransactionsBoard.qml

```diff
@@ -21,7 +22,216 @@ Rectangle {
 
     Controls.DiodeHeader {
         id: diodeHeader
-        text: qsTr("TRANSACTIONS")
+        text: qsTr("NODE TRANSACTIONS")
         menuLabelText: qsTr("Complete View")
     }
-}
\ No newline at end of file
+    TableView {
```

Comment:
Is this a copy-paste of `XCITE/Home/TransactionTable`? If so you can basically just import `Home` and replace all of it with a single-liner. Should be able to customise its interface for the most part via parameters